project/Build.scala

```diff
@@ -131,7 +131,7 @@ object Build extends sbt.Build {
       val AsyncAwait = "0.9.6-RC2"
       val ScalaTest = "2.1.7"
       val MacWire = "2.2.2"
-      val Marathon = "1.2.0-RC1"
+      val Marathon = "1.3.0"
```

Comment:
Why not 1.3.13?

Reply:
It should eventually be 1.3.13. I think it makes sense to first have everything adjusted to 1.3.0 and compiling, and then bump further.

As a third step I'd be inclined to bump Play to 2.5.18, and eventually to 2.6